docs/source/guides/writer/chapters/integrating.rst

```diff
@@ -0,0 +1,45 @@
+Integrating Avocado
+===================
+
+Coverage.py
+~~~~~~~~~~~
+
+Testing software is important, but knowing the effectiveness of the tests,
+like which parts are being exercised by the tests, may help develop new tests.
+
+`Coverage.py`_ is a tool designed for measuring code coverage of Python
+programs. It runs monitoring the program's source, taking notes of which
+parts of the code have been executed.
+
+It is possible to use Coverage.py while running Avocado tests. As Avocado
+spawn sub-processes to run the tests, the `concurrency` parameter should be
+set to `multiprocessing`.
+
+To make the Coverage.py parameters visible to other processes spawned by
+Avocado, create the `.coveragerc` file in the project's root folder.
+Following is an example::
+
+    [run]
+    concurrency = multiprocessing
+    source = foo/bar
+    parallel = true
+
+According to the documentation of Coverage.py, when measuring coverage in
+a multi-process program, setting the `parallel` parameter will keep the data
+separate during the measurement.
+
+With the `.coveragerc` file set, one possible workflow to use Coverage.py to
```

Comment:
.coveragerc needs double backtick ``.coveragerc`` to be nicely displayed in github.